CONFIG.md

```diff
@@ -4704,9 +4704,24 @@ Lua language server.
 **By default, lua-language-server doesn't have a `cmd` set.** This is because nvim-lspconfig does not make assumptions about your path. You must add the following to your init.vim or init.lua to set `cmd` to the absolute path ($HOME and ~ are not expanded) of you unzipped and compiled lua-language-server.
 
 ```lua
-local sumneko_root_path = "/path/to/lua-language-server"
+local system_name
+if vim.fn.has("mac") == 1 then
+  system_name = "macOS"
+elseif vim.fn.has("unix") == 1 then
+  system_name = "Linux"
+elseif vim.fn.has('win32') == 1 then
+  system_name = "Windows"
+else
+  print("Unsupported system for sumneko")
+end
+
+-- If you previously installed via the now deprecated LspInstall, you can set 
```

Comment:
obsolete comment line?

Reply:
Yep, still pushing

Reply:
will shut up then :)

Reply:
No please don't! Your commentary is valuable :)

Reply:
Should be up to date now

Reply:
Looks good now! (I have no idea if the Windows check is sufficient -- there's also `win64` and `wsl` -- and whether sumneko even supports BSD, so let's wait for complaints there ;))

Reply:
The win32 should cover win64, as for the bsds... they can handle modifying a couple lines of lua :P